app/src/main/java/com/example/frankfurter/Core.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.frankfurter
+
+import android.content.Context
+import com.example.data.core.ProvideResources
+import com.example.presentation.core.ClearViewModel
+import com.example.presentation.core.RunAsync
+import com.example.presentation.main.Navigation
+
+interface Core {
+
+    fun navigation(): Navigation.Mutable
+
+    fun runAsync(): RunAsync
+
+    fun clearViewModel(): ClearViewModel
+
+    fun provideResources(): ProvideResources
+
+    fun context(): Context
```

Comment:
fun provideCacheModule() : CacheModule


interface CacheModule { //todo put to data module

fun database() : CurrenciesDataBase

class Base(private val context : Context ) : CacheModule {


private val database by lazy {
Room.dataBaseBuilder(context)